### DISCUSSION TITLE
Can schemacrawler generate ERD based on input sql file?

### DISCUSSION THREAD

**Discussion by ashwini-desai** [schemacrawler/SchemaCrawler#819] [Q&A] [answered]
I have a use case to auto generate ERD during CI. For this, I already have schema sql file generated.
Can I pass this as input to schemacrawler? I want to do this without active database connection.

**Comment by sualeh** [comment, marked as answer]
@ashwini-desai No, SchemaCrawler is designed to run against a live database or an [offline snapshot](https://www.schemacrawler.com/offline.html). You could consider spinning up a Docker container with your database during the CI build and initialize with your script. Then you can spin SchemaCrawler also in a Docker container and run against the database.

What database are you using, and what CI/ CD system?
